estuary/ui
# Check for entity existence before entering edit workflow

## Changes

The following features are included in this PR:

1. Create an _Entity Not Found_ page.

1. Create an entity existence guard that extracts a live spec ID in the URL and evaluates its presence in the `live_specs` table.

## Tests

Approaches to testing are as follows:

1. Copy the edit URL for an existing entity, delete the entity, then navigate to the edit URL and observe the _Entity Not Found_ error page.

1. Enter the edit workflow for an existing entity and confirm that the workflow has not changed.

## Issues

The issues directly below are completely resolved by this PR:
#413 

## Content

_Were there any changes to [content](https://github.com/estuary/ui/tree/main/src/lang) that you need to get reviewed?_

## Screenshots

**Entity Not Found Error Page**

<img width="1059" alt="pr_screenshot-461-deleted_entity_edit_url-entity_not_found_page" src="https://user-images.githubusercontent.com/77648584/214071426-b2209a67-a701-44f1-91ed-2d8d1269dac9.PNG">
